library/core/src/sync/atomic.rs

```diff
@@ -2685,61 +2780,63 @@ macro_rules! atomic_int {
                 unsafe { &mut *(v as *mut $int_type as *mut Self) }
             }
 
-            #[doc = concat!("Get non-atomic access to a `&mut [", stringify!($atomic_type), "]` slice")]
-            ///
-            /// This is safe because the mutable reference guarantees that no other threads are
-            /// concurrently accessing the atomic data.
-            ///
-            /// # Examples
-            ///
-            /// ```
-            /// #![feature(atomic_from_mut)]
-            #[doc = concat!($extra_feature, "use std::sync::atomic::{", stringify!($atomic_type), ", Ordering};")]
-            ///
-            #[doc = concat!("let mut some_ints = [const { ", stringify!($atomic_type), "::new(0) }; 10];")]
-            ///
-            #[doc = concat!("let view: &mut [", stringify!($int_type), "] = ", stringify!($atomic_type), "::get_mut_slice(&mut some_ints);")]
-            /// assert_eq!(view, [0; 10]);
-            /// view
-            ///     .iter_mut()
-            ///     .enumerate()
-            ///     .for_each(|(idx, int)| *int = idx as _);
-            ///
-            /// std::thread::scope(|s| {
-            ///     some_ints
-            ///         .iter()
-            ///         .enumerate()
-            ///         .for_each(|(idx, int)| {
-            ///             s.spawn(move || assert_eq!(int.load(Ordering::Relaxed), idx as _));
-            ///         })
-            /// });
-            /// ```
+            // #[doc = concat!("Get non-atomic access to a `&mut [", stringify!($atomic_type), "]` slice")]
```

Comment:
This sould work with `concat!` and `stringify!`, no? They are helpful for the other PR as well, so let's add them I'd say.